docs/user_guide/nwbconverter.rst

```diff
@@ -44,21 +44,30 @@ keys of``data_interface_classes``.
 
 This creates an :py:class:`.NWBConverter` object that can aggregate and distribute across
 the data interfaces. To fetch metadata across all of the interfaces and merge
-them together, call::
+them together, call.
```

Comment:
As per discussion on the backend configuration tutorial, I thought we decided on `:` prior to steps using example code blocks?

So for consistency with https://neuroconv.readthedocs.io/en/main/user_guide/backend_configuration.html

Which I think was justified from `pandas` documentation?

Reply:
I was mostly trying to get rid of using the `::` method of starting a code snippet, since this does not Python-colorize the code. We can change back to using a single : though (which has no formatting effect). I took a look through the scikit learn docs, which is my favorite library to base style decisions on. It looks like they *almost* always precede a code snippet with ":". I'll change it back in a follow-up PR.